src/ngx_http_upsync_module.c

```diff
@@ -2527,14 +2567,18 @@ ngx_http_upsync_send_handler(ngx_event_t *event)
     if (upsync_type_conf->upsync_type == NGX_HTTP_UPSYNC_ETCD) {
         if (upsync_server->index != 0) {
             ngx_sprintf(request, "GET %V?wait=true&recursive=true&waitIndex=%d"
-                        " HTTP/1.0\r\nHost: %V\r\nAccept: */*\r\n\r\n", 
+                        " HTTP/1.0\r\nHost: %V\r\nAccept: */*\r\n"
+                        "%V\r\n", 
```

Comment:
Got！Is it a little tricky? Adding a judgement 'if upscf->upsync_basic_auth.len > 0' will be more clear? Do you think?